### DISCUSSION TITLE
WikiJS VIOLATES privacy laws (such EU GDPR)

### DISCUSSION THREAD

**Discussion by Nolram12345** [requarks/wiki#7410] [Error / Bug Report]
Currently, WikiJS gives no way for users to delete their own account. This is a violation of almost every data privacy protection law in most countries.

This feature needs to be added to WikiJS as soon as possible, as otherwise every WikiJS user could be held legally accountable.

**Comment by fnafox** [comment]
IANAL but afaik GDPR Art 15 doesn't provide implementation requirements, just that account erasure must be provided upon request. If you have an accessible administrator contact where a user can request their data to be deleted, you are not in violation.
